server/core/server.py

```diff
@@ -42,7 +41,6 @@ def on_auth_error(request: Request, exc: Exception):
 
 def init_routers(app_: FastAPI) -> None:
     app_.include_router(router)
-    app_.include_router(ee_router)
```

Comment:
What's the reason to remove ee_router?

Reply:
Added back, must be by mistakenly got removed.